framework/filters/AccessRule.php

```diff
@@ -136,19 +142,26 @@ protected function matchController($controller)
      */
     protected function matchRole($user)
     {
-        if (empty($this->roles)) {
+        $privileges = empty($this->roles) ? [] : $this->roles;
```

Comment:
`$privileges` could be renamed to `$items` to be consistent with RBAC implementation.